apollo-gradle-plugin/src/main/groovy/com/apollostack/android/gradle/ApolloIRGenTask.groovy

```diff
@@ -40,7 +42,8 @@ public class ApolloIRGenTask extends NodeTask {
     dependsOn(ApolloCodeGenInstallTask.NAME)
 
     possibleGraphQLPaths = buildPossibleGraphQLPaths()
-    outputDir = project.file("src/$variant/graphql/generatedIR")
+    schemaFile = userProvidedSchemaFile() ?: searchForSchemaFile()
+    outputDir = schemaFile.getParentFile()
```

Comment:
So the implications of having IR file under `src` folder is that it will be under source control. I don't think this is good idea as it will be a big file that probably won't be possible to review before committing to source control. So I would move it to `build` folder